config/wpcalypso.json

```
@@ -61,6 +61,7 @@
 		"help/gpt-response": true,
 		"home/layout-dev": true,
 		"hosting/datacenter-picker": true,
+		"hosting-server-settings-enhancements": false,
```

```suggestion
		"hosting-server-settings-enhancements": true,
```
See my other comment